src/k8s/pkg/k8sd/setup/kube_apiserver.go

```diff
@@ -13,6 +14,7 @@ import (
 
 type apiserverAuthTokenWebhookTemplateConfig struct {
 	URL string
+	CA  string
```

Comment:
Let's call this `CAPath` to make it clear that we expect a file path instead of the certificate itself.